### PR TITLE
deprecate direct integration

### PR DESCRIPTION
**Pull request by michaelckelly**
- Deprecate direct integration as it is no longer supported
- Remove `/item/delete` in favor of `/item/remove`

To be published as a major version bump given the breaking change

### CODE REVIEW

File: src/test/java/com/plaid/client/integration/AbstractItemIntegrationTest.java [hunk 1]
```diff
@@ -43,22 +49,31 @@ protected static void assertAccount(Account actualAccount,
...
-          .withCredentials("password", "pass_good")).execute();
+    Response<SandboxPublicTokenCreateResponse> createResponse =
+      client().service().sandboxPublicTokenCreate(new SandboxPublicTokenCreateRequest(setupItemInstitutionId(), setupItemProducts())).execute();
+    assertSuccessResponse(createResponse);
+    Response<ItemPublicTokenExchangeResponse> response =
+      client().service().itemPublicTokenExchange(new ItemPublicTokenExchangeRequest(createResponse.body().getPublicToken())).execute();
     assertSuccessResponse(response);
-    itemCreateResponse = response.body();
+
+    exchangeTokenResponse = response.body();
```
Comment (davidzhanghp) [inline src/test/java/com/plaid/client/integration/AbstractItemIntegrationTest.java]: can we add this. in front of setting these fields

File: src/test/java/com/plaid/client/integration/TransactionsGetTest.java [hunk 2]
```diff
@@ -50,9 +51,18 @@ public void testSuccess() throws Exception {
...
 
+    Response<TransactionsGetResponse> response = null;
+    for (int i = 0; i < 5; i++) {
+      response = client().service().transactionsGet(request).execute();
+      if (response.isSuccessful()) {
+        break;
+      } else {
+        ErrorResponse errorResponse = client().parseError(response);
+        assertEquals(errorResponse.getErrorCode(), "PRODUCT_NOT_READY");
+        Thread.sleep(5000);
```
Comment (davidzhanghp) [inline src/test/java/com/plaid/client/integration/TransactionsGetTest.java]: can we lower this number?

File: src/test/java/com/plaid/client/integration/AbstractItemIntegrationTest.java (hunk 1 above)
Comment (michaelckelly) [reply src/test/java/com/plaid/client/integration/AbstractItemIntegrationTest.java]: :zap:

File: src/test/java/com/plaid/client/integration/TransactionsGetTest.java (hunk 2 above)
Comment (michaelckelly) [reply src/test/java/com/plaid/client/integration/TransactionsGetTest.java]: Will try with 3000

**Pull request merged**